types/scene/wlr_scene.c

```diff
@@ -2134,6 +2053,34 @@ static void scene_entry_render(struct render_list_entry *entry, const struct ren
 			});
 		}
 		break;
+	case WLR_SCENE_NODE_BLUR_SOURCE:;
```

Comment:
I think we miss using the `transparency_mask_source` here, but I can add that bit to this branch if it's being a pain